org.lflang/src/org/lflang/generator/ts/TSReactionGenerator.kt

```diff
@@ -103,11 +107,20 @@ class TSReactionGenerator(
         reaction: Reaction,
         reactPrologue: String,
         reactEpilogue: String,
-        reactFunctArgs: StringJoiner,
+        reactFuncArgs: StringJoiner,
         reactSignature: StringJoiner
     ): String {
         // Assemble reaction triggers
+        var reactCode = ""
+        var reactDeadline = "}"
         val reactionTriggers = StringJoiner(",\n")
+
+        if (!reaction.isMutation()) {
```

Comment:
Can mutations not have deadlines? I think this can all go away and we can just reinstate the original line `139`.

Reply:
You're right. I deleted this part. https://github.com/lf-lang/lingua-franca/pull/1318/commits/f327feef4c499641d83833eb1fd3591f20c35834